Xamarin.Forms.Core/Cells/CheckBoxCell.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+
+namespace Xamarin.Forms
+{
+	public class CheckBoxCell : Cell
```

Comment:
Drop the new Cell types; folks should be using ViewCells with DataTemplates.

Reply:
<3 <3 no? All about that TableView :) i can drop it though if it is a requirement.